examples/demo_app/db/schema.rb

```diff
@@ -10,6 +10,6 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_09_01_193936) do
+ActiveRecord::Schema.define(version: 0) do
```

Comment:
New Rails 6 apps don't need the extra active_storage migration, which explains this change.